apps/web/src/pages/Applications/ApplicationSuccessPage/components/Content.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { ExternalLink, Link } from "@gc-digital-talent/ui";
+import { useLocale } from "@gc-digital-talent/i18n";
+
+import useRoutes from "~/hooks/useRoutes";
+
+const DefaultContent = () => {
```

Comment:
Does this file get used?  It doesn't export anything and it seems like the copy is inline in the other file?

Reply:
Oops, yeah this was left over from my first attempt at doing this and it accidentally got committed 😛 

[remove unused file](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6589/commits/1117e5cfc3e61745a613949e74ad5cef610ad9b9)